runtime/compiler/env/VMJ9.h

```diff
@@ -975,8 +986,13 @@ class TR_J9VMBase : public TR_FrontEnd
    virtual bool acquireClassTableMutex();
    virtual void releaseClassTableMutex(bool);
 
+#if defined(JITSERVER_SUPPORT)
```

Comment:
This is needed in `J9TransformUtil.cpp` and `J9ValuePropagation.cpp`, so let's leave it outside #ifdef.